src/Symfony/Component/Security/Core/Tests/User/UserTest.php

```diff
@@ -123,4 +123,13 @@ public function testEraseCredentials()
         $user->eraseCredentials();
         $this->assertEquals('superpass', $user->getPassword());
     }
+
+    /**
+     * @covers Symfony\Component\Security\Core\User\User::__toString
```

Comment:
If possible, we should remove this `@covers` annotation because Symfony no longer uses it in its tests.